CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v10.0.1](https://github.com/voxpupuli/puppet-php/tree/v10.0.1) (2023-10-28)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-php/compare/v10.0.0...v10.0.1)
+
+**Implemented enhancements:**
```

Comment:
According to semver, this should be a minor version bump: 10.1.0:
https://semver.org/

Reply:
I'll open a new PR for 10.1.0 shortly.